modules/identity-provider-about-request-header.adoc

```diff
@@ -9,6 +9,11 @@ A request header identity provider identifies users from request
 header values, such as `X-Remote-User`. It is typically used in combination with
 an authenticating proxy, which sets the request header value.
 
+[IMPORTANT]
```

Comment:
nit: move this to the place of the original warning - under "For example: [x-]`https://www.example.com/auth-proxy/oauth/authorize?${query}`"